server/api/applicant.js

```diff
@@ -90,10 +106,19 @@ function applicant({ app, db, pgp }) {
     ON term.id = termapplication.term
     WHERE term.visible = true
     */
-  app.get("/api/applicant/applications/available", async (req, res) => {
-    const ret = await getAvailableApplications(req, res);
-    res.json(ret);
-  });
+  app.get(
+    "/api/applicant/applications/available",
+    AS(async (req, res) => {
+      getAvailableApplications(req, res)
+        .then((ret) => {
```

Comment:
Is there a reason why you are converting an await to a then? If it is to get the catch, you can do it with a try/catch block. I won't block on this, just curious.